sdk/cosmosdb/cosmos/src/client/Item/ItemResponse.ts

```diff
@@ -12,9 +13,10 @@ export class ItemResponse<T extends ItemDefinition> extends ResourceResponse<T &
     headers: CosmosHeaders,
     statusCode: number,
     subsstatusCode: number,
-    item: Item
+    item: Item,
+    diagnostics: CosmosDiagnostics
   ) {
-    super(resource, headers, statusCode, subsstatusCode);
+    super(resource, headers, statusCode, diagnostics, subsstatusCode);
```

Comment:
just a suggestion: Can we add new argument towards the end?

Reply:
I agree it looks a bit awkward to have diagnostics in between statusCode and subsstatusCode, but the reason I put in between is because `diagnostics` parameter will be changed to `non-optional` in next PR, and you can't mix optional and non-optional fields.